modules/metering-install-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-prerequisites_{context}"]
+= Prerequisites
+
+Operator Metering requires the following components:
+
+* A StorageClass for dynamic volume provisioning. Metering supports a number of different storage solutions. See xref:../metering/metering-configuring-metering.adoc#configuring-storage[configuring storage for metering] for more details.
+* A Prometheus installation within the cluster configured to do Kubernetes cluster-monitoring. Openshift 4.x clusters have Prometheus enabled by default on installation.
+* 4GB Memory and 4 CPU Cores available cluster capacity.
+* The minimum resources needed for the largest single pod installed by metering is 2GB of memory and 2 CPU cores.
+** Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+* A properly configured kubectl to access the Kubernetes cluster.
```

Comment:
Recommend saying the `oc` CLI instead of kubectl. And removing "Kubernetes"

Reply:
Im not sure we even need this line. It only was added because some people in the past had issues cause their kubectl wasn't setup to the correct cluster.